### PR TITLE
Make extreme rays great again.

### PR DESCRIPTION
**Pull request by syarra**

### CODE REVIEW

**Comment by rmtfleming** [comment]
Nice

File: test/verifiedTests/testTopology/runLrsBash.m [hunk 1]
```diff
@@ -0,0 +1,29 @@
+function runLrsBash(filename, positivity, inequality)
```
Comment (laurentheirendt) [inline test/verifiedTests/testTopology/runLrsBash.m]: Can we not combine `runLrsBash` and `runLrs`?

Comment (syarra) [inline test/verifiedTests/testTopology/runLrsBash.m]: Maybe!

Comment (syarra) [inline test/verifiedTests/testTopology/runLrsBash.m]: Done!

**Comment by laurentheirendt** [comment]
Great job @syarra! :cactus:

**Pull request merged**